src/views/HireUs.vue

```diff
@@ -0,0 +1,337 @@
+<template>
+  <section
+    id="hire-us"
+    aria-label="Hire Us Section"
+    class="grid place-items-center"
+  >
+    <header class="text-center mb-20">
+      <div></div>
+      <h1 class="font-bold py-6 text-3xl">Hire Us</h1>
+    </header>
+
+    <div
+      id="members-cards-info"
+      class="w-96 bg-purple-400 p-6 rounded-xl relative text-blue-700 z-[2] 
+      left-0  md:left-[-14rem] border-white border-1 h-[80%]"
+    >
+      <div class="card relative">
+        <header class="text-2xl font-semibold">Our Team members</header>
+        <section class="members flex flex-col items-center justify-center gap-2 
+         ">
+          <div>
+            <h2 class="font-bold text-2xl">Sam T</h2>
+            <img
+              src="/programmers/skman-tyler.svg"
+              alt="izak tarashandegan the founder and coder"
+              class="float-right w-1/2"
+            />
+            <h3 class="text-lg font-semibold">Lead Coder</h3>
+            <a href="#sam-info" class="text-blue-700 underline">more info...</a>
+          </div>
+          <div>
+            <h2 class="font-bold text-2xl">Izak T</h2>
+            <img
+              src="/programmers/skman-tyler.svg"
+              alt="izak tarashandegan the founder and coder"
+              class="float-right w-1/2"
+            />
+            <h3 class="text-lg font-semibold">Founder / Coder</h3>
+            <a href="#izak-info" class="text-blue-700 underline">more info...</a>
+          </div>
+        </section>
+      </div>
+
+      <section
+        class="contact-info absolute  grid grid-cols-1 md1:grid-cols-2 max-w-sm md:max-w-xl lg:max-w-3xl p-6 mb-16 md:mb-8 bg-white border border-gray-200 rounded-lg shadow hover:bg-gray-100 glass"
+      >
+        <section class="col-span-1">
+          <div class="card">
+            <div class="font-normal text-gray-900">
+              <h2
+                class="text-center w-full pr-[28px] font-semibold text-2xl text-blue-700"
+              >
+                Contact Us to make your dream a reality
+              </h2>
+              <div class="flex flex-col gap-5 mt-16">
+                <div>
+                  Tel:
+                  <a href="tel:+1 (424)-442-0591">(424)-442-0591 </a>
+                </div>
+                <div>
+                  Email:
+                  <a href="mailto:pazpaz25@gmail.com">pazpaz25@gmail.com</a>
+                </div>
```

Comment:
You should probably get a domain for this site.  google domains moved to squarespace. https://domains.squarespace.com/google-domains. 30usd per year is a pretty good deal. some are much cheaper.

Reply:
you can then forward emails from your domain to your personal email, or a new email - https://support.squarespace.com/hc/en-us/articles/19000909092237-Email-forwarding-with-a-Squarespace-domain

Reply:
I believe to talk about it and come with good name that has IT in it